spring-cloud-aws-autoconfigure/src/main/java/io/awspring/cloud/autoconfigure/core/CredentialsProviderAutoConfiguration.java

```diff
@@ -45,6 +49,9 @@
 @EnableConfigurationProperties(CredentialsProperties.class)
 public class CredentialsProviderAutoConfiguration {
 
+	public static final String STS_WEB_IDENTITY_TOKEN_FILE_CREDENTIALS_PROVIDER = "software.amazon.awssdk.services.sts.auth.StsWebIdentityTokenFileCredentialsProvider";
+	public static final String AWS_ROLE_ARN_ENV_VARIABLE = "AWS_ROLE_ARN";
```

Comment:
We should modify `CredentialsProperties` with nested `StsProperties` and move all of this there.